packages/rum/src/domain/record/observers.ts

```diff
@@ -497,3 +498,10 @@ export function initFrustrationObserver(lifeCycle: LifeCycle, frustrationCb: Fru
     }
   }).unsubscribe
 }
+
+function getEventTarget(event: Event): Node {
+  if (event.composed === false) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```ts
if (!event.composed) { ... }
```
this way, it would still use `event.target` for emulated events without a `composed` property